app/views/submissions/_assigned_evaluators.html.erb

```diff
@@ -57,7 +59,7 @@
                     <%= evaluator_score(assignment).formatted_score %>
                   </span>
                 </td>
-                <td data-label="">
+                <td data-label="Action">
                   <div class="display-flex flex-wrap">
                     <% if assignment&.evaluation&.completed_at %>
                       <%= link_to(revision_evaluation_path(assignment.evaluation)) do %>
```

Comment:
this button moved to a link under the score on figma..
<img width="877" alt="Screenshot 2025-03-04 at 10 07 04 PM" src="https://github.com/user-attachments/assets/29e47eb3-dc3a-4be9-8755-81e9b6c9b391" />